src/locales/en.json

```diff
@@ -41,6 +41,12 @@
 		"messageResetPermission": "Something <strong>not working properly with the notifications</strong>, please <reloadButton>reload this page</reloadButton>.",
 		"messageOffline": "You are currently <strong>offline. Some features may not be available.</strong>"
 	},
+	"loginState": {
+		"confirm": "Confirm",
+		"message": "You are currently in an <strong>active session</strong>. Use your passkey to continue and fetch your credential in your wallet.",
```

Comment:
Hm, is the session **active**? Navigating to `/?state=...` with an active session skips the page entirely, so I think this should rather say something like "expired session" or "suspended session"?

Reply:
You're right. We were trying to hint that the logged in user is already cached because issuing a new VC and being navigated to a new tab where we had to login again wasn't really user friendly. Maybe 'Hi <name>" is enough though, so we should drop the first sentence completely